frontend/app/src/App.vue

```diff
@@ -76,7 +76,7 @@
         @visible:update="help = $event"
         @about="showAbout = true"
       />
-      <v-main v-if="logged">
```

Comment:
Is there a reason why this check was removed?

Reply:
This is redundant, already checked at line 4

Reply:
Oh, nice catch, thanks. It is always nice to have a second pair of eyes. Sometimes it is really easy to miss stuff.